scout/ecm_prep.py

```diff
@@ -139,6 +139,73 @@ def update_active_measures(cls,
         return run_setup
 
 
+def add_internal_gains_aggregate(msegs: dict, years, ig_names=None, new_name="internal gains"):
```

Comment:
I think this is better suited to be done in the `ResTLoads_Process.py` file

Reply:
Though I am a little confused, there is no "people" component load in ResStock right? So is this just for the original component loads?